src/Tools/BuildActionTelemetryTable/CodeActionDescriptions.cs

```diff
@@ -0,0 +1,432 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+using System.Collections.Generic;
+
+namespace BuildActionTelemetryTable;
+
+internal static class CodeActionDescriptions
+{
+    public static ImmutableDictionary<string, string> CodeActionDescriptionMap { get; } = new Dictionary<string, string>()
```

Comment:
The tool will now generate a new version of this file with "auto-generated" descriptions. It encourages you to review the descriptions before updating.